examples/demo-app/now.json

```diff
@@ -0,0 +1,7 @@
+{
+  "static": {
+      "rewrites": [
```

Comment:
I don't know if it will autodetect this, but you might have to specify that this is now v1